src/components/Footer.js

```diff
@@ -29,14 +29,14 @@ export default class Footer extends Component {
     const { socials } = this.props
     return (
       <footer className="Footer">
-        <Col sm={4} className="Footer-element">
```

Comment:
can we add a layer of `<div class="container">` here so it doesn't go full browser width?